petl/util/timing.py

```diff
@@ -9,9 +11,9 @@
 from petl.util.statistics import onlinestats
 
 
-def progress(table, batchsize=1000, prefix="", out=sys.stderr):
+def progress(table, batchsize=1000, prefix="", out=None):
     """
-    Report progress on rows passing through. E.g.::
+    Report progress on rows passing through to a log. E.g.::
```

Comment:
Suggest 'Report progress on rows passing through to a file or file-like object (defaults to ``sys.stderr``).'